apps/jupyter/extensions/notebook-runner/src/iframe-api.ts

```diff
@@ -179,7 +179,7 @@ export class EmbeddedPythonCallbacks {
       );
 
       this.appTranslator.displayErrorFromException(MessageKeys.ExportError, e);
-      throw e;
+      return undefined;
```

Comment:
As discussed on teams, we can throw and propagate the error message to the caller :)